android/src/main/java/com/mapbox/mapboxgl/Convert.java

```diff
@@ -219,6 +219,14 @@ static void interpretMapboxMapOptions(Object o, MapboxMapOptionsSink sink) {
     if (myLocationEnabled != null) {
       sink.setMyLocationEnabled(toBoolean(myLocationEnabled));
     }
+    final Object myLocationTrackingMode = data.get("myLocationTrackingMode");
+    if (myLocationTrackingMode != null) {
+      sink.setMyLocationTrackingMode(toInt(myLocationTrackingMode));
+    }
+    final Object myLocationVerticalAlignment = data.get("myLocationVerticalAlignment");
```

Comment:
what does myLocationVericalAligment mean? how the location is vertically aligned on screen?
this concept doesnt' exist in the Android SDK or iOS afaik. We use the concept of ltrb paddding.

Reply:
Thanks for your comment! I removed the code. Let's do lorn instead.